cmd/ui/src/utils.ts

```diff
@@ -209,7 +215,7 @@ export const transformToFlatGraphResponse = (graph: GraphResponse) => {
                 nodetype: value.kind,
                 name: value.label,
                 objectid: value.objectId,
-                system_tags: value.isTierZero ? 'admin_tier_0' : undefined,
+                system_tags: tags,
```

Comment:
This value should be a string as that is the way the system_tags property is stored in the database. A quick fix to this can be made by joining the array elements, i.e., `tags.join(' ')`